Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst

```diff
@@ -172,19 +172,19 @@ Command keywords and values
          Value is an uid of the :php:`sys_language` to localize the record into.
          Basically a localization of a record is making a copy of the record
          (possibly excluding certain fields defined with :php:`l10n_mode`) but
-         changing relevant fields to point to the right sys language / original
+         changing relevant fields to point to the right :php:`sys_language` / original
          language record.
 
          Requirements for a successful localization is this:
 
          - :code:`[ctrl]` options "languageField" and "transOrigPointerField" must be
            defined for the table
 
-         - A :code:`sys_language` record with the given :code:`sys_language_uid` must
+         - A :php:`sys_language` record with the given :php:`sys_language_uid` must
            exist.
 
          - The record to be localized by currently be set to "Default" language
-           and not have any value set for the :code:`transOrigPointerField` either.
+           and not have any value set for the :php:`transOrigPointerField` either.
```

Comment:
I'd prepend "TCA" (as word, not code) in front of the `transOrigPointerField` here.

Reply:
How should that be: ```:php:TCA:transOrigPointerField```? (with single quotes around `transOrigPointerField`)

Reply:
No, like this:

```
... for the TCA :php:`transOrigPointerField` ...
````